### ISSUE TITLE
File transmission requirements: use Ctrl+C & Ctrl+V

### ISSUE THREAD

**Issue by iminifly** [kingToolbox/WindTerm#1239]
This software is amazing. Its functions are very comprehensive and the UI is very beautiful;
However, when I upload files, I can only drag them or use the right-click menu, which is not very convenient; I expect to support the mode of Ctrl+C Ctrl+V in future versions

**Comment by nopaben** [comment]
> This software is amazing. Its functions are very comprehensive and the UI is very beautiful; However, when I upload files, I can only drag them or use the right-click menu, which is not very convenient; I expect to support the mode of Ctrl+C Ctrl+V in future versions

and , only  upload can use drag , download files only use right-click menu, I expect to support drag download.

**Comment by bequt** [comment]
Yes, it's really recommended in the file browser to be able to use shortcuts to copy and paste for file transfers。

**Comment by yang1161519465** [comment]
It would be great to support this feature. It's extremely useful, especially for users with limited screen space. My screen isn't large enough to display multiple windows comfortably, so this feature would significantly improve usability and productivity.
